common/src/main/java/com/microsoft/identity/common/internal/cache/MicrosoftStsAccountCredentialAdapter.java

```diff
@@ -76,7 +80,11 @@ public AccessTokenRecord createAccessToken(
             accessToken.setCredentialType(CredentialType.AccessToken.name());
             accessToken.setHomeAccountId(SchemaUtil.getHomeAccountId(clientInfo));
             accessToken.setRealm(getRealm(strategy, response));
-            accessToken.setEnvironment(strategy.getIssuerCacheIdentifier(request));
+            if (!StringUtil.isEmpty(response.getAuthority())) {
```

Comment:
Is there a case where the authority is expected to be null on the response? Or just being defensive here?